internal/pkg/chain/sampler.go

```diff
@@ -48,7 +61,8 @@ func (s *Sampler) Sample(ctx context.Context, head block.TipSetKey, epoch abi.Ch
 	}
 
 	buf := bytes.Buffer{}
-	buf.Write(ticket.VRFProof)
+	vrfDigest := blake2b.Sum256(ticket.VRFProof)
```

Comment:
By request of @sternhenri , https://github.com/filecoin-project/go-filecoin/pull/3800#discussion_r383325303